addons/notes/src/Panel.tsx

```diff
@@ -10,13 +10,14 @@ import Markdown from 'markdown-to-jsx';
 
 import { PARAM_KEY, API, Parameters } from './shared';
 
-const Panel = styled.div({
+const Panel = styled.div(props => ({
```

Comment:
@vpicone take a look at [build_test_deploy check](
https://circleci.com/gh/storybooks/storybook/105690?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link), you must type `props` (not sure we can do better than `any`) or stick to your previous version with object destructuring `({theme}) => ({ ... })`